stacks/container-apps/exchange.prx.org.yml

```diff
@@ -102,7 +108,17 @@ Parameters:
     Default: web
   HealthCheckPath:
     Type: String
-    Default: /api/v1
+    Default: /ping
```

Comment:
Not sure if this is the time to address it ... but this stack feels kind of strange with all the internal-only params that are only looked up in 1 place.